uint32 toID
uint32 fromID
#bool getAll
---
#float32[] distances
float32 distance